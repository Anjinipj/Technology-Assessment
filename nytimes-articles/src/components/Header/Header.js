import "./Header.css";

const Header = () => {
  return (
    <header>
      <h1>The Most Popular Times Articles</h1>
    </header>
  );
};

export default Header;
