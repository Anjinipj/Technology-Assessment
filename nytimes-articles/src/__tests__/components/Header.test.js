import React from 'react';
import renderer from 'react-test-renderer';

import Header from '../../components/Header/Header';

it('renders correctly Header ', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});