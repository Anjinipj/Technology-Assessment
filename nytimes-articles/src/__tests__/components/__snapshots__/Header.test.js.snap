// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Header  1`] = `
<header>
  <h1>
    The Most Popular Times Articles
  </h1>
</header>
`;
